src/generate.rs

```diff
@@ -204,33 +204,33 @@ async fn create_xmltv(
             String::from("\n      <icon src=\"\"></icon>")
         };
 
-        let description = game.description().await.unwrap_or_else(|| String::from(""));
+        let mut description = game.description().await.unwrap_or_else(|| String::from(""));
+        if description == "" {
```

Comment:
You should use `.is_empty()` to check if a string is empty. This is a common lint that can be caught using clippy. `cargo clippy`

Reply:
Since `.description()` returns an option, we can actually just set the default value during the "unwrap_or_else" instead of setting it to blank, then evaluating the blank

```diff
-        let mut description = game.description().await.unwrap_or_else(|| String::from(""));
-        if description == "" {
-            description = format!("Watch the {} take on the {}.",
-                                  game.away_team.team_name, game.home_team.team_name);
-        }
+        let description = game.description().await.unwrap_or_else(|| {
+            format!(
+                "Watch the {} take on the {}.",
+                game.away_team.team_name, game.home_team.team_name
+            )
+        });
```

Reply:
I tried this new change but it resulted in `<desc lang="en"></desc>`

Reply:
Hmm, must be an empty string coming across from the API. I'm fine leaving it as is then! I'll go ahead and merge this.